examples/transformers/whisper.py

```diff
@@ -7,8 +8,26 @@
 audio = requests.get("https://www.nasa.gov/62283main_landing.wav").content
 
 task = "automatic-speech-recognition"
+architecture = "openai/whisper-small"
```

Comment:
```suggestion
architecture = "openai/whisper-tiny"
```

Can we use `whisper-tiny` for faster execution?

Reply:
Yep